banking/urls.py

```diff
@@ -1,13 +1,15 @@
 from django.urls import path
-from .views import index, employee
+from .views import index, employee, customer
 
 
 app_name = 'banking'
 
 urlpatterns = [
     path('', index.index, name='index'),
     path('employee/', employee.index, name='employee'),
+    path('customer/', customer.index, name='customer'),
```

Comment:
not important, but would be nice if endpoints were grouped, perhaps alphabetically